src/controller/artifact/schema/v1/schema.go

```diff
@@ -0,0 +1,33 @@
+package v1
+
+const (
+	// XHarborAttributes is the specific key defined in artifact config layer
+	// for user-defined artifact to configure XHarborAttribute JSON schema
+	XHarborAttributes = "xHarborAttributes"
+	// SchemaVersionV1 1
+	SchemaVersionV1 = 1
+)
+
+// Schema define a JSON schema for default processor to extract artifact layer data
+// to artifact model in a specific way
```

Comment:
give a example json in the comments?

Reply:
There are example json in unittest. I will add example in comments.